modules/nf-core/bbmap/repair/tests/nextflow.config

```diff
@@ -0,0 +1,6 @@
+process {
+    withName: BBMAP_REPAIR {
+        ext.args = params.module_args
+        cpus     = { 2 * task.attempt }
```

Comment:
Why is this line here?

Reply:
Is it wrong?

Reply:
Ah yes, I wanted to pass the number of threads explicitly to tests but I didn't. Should I remove it?

Reply:
The tests set the number of threads automatically, so shouldn't need this here.

Reply:
Ok I've literally deleted only that line and tests are failing because CI seems does not work. It's frustrating 😓

Reply:
> Ok I've literally deleted only that line and tests are failing because CI seems does not work. It's frustrating 😓

Yeah, that is not your fault, the docker tests seem to have broken for some reason :| 
Hopefully that'll get fixed soon as those who understand the CI get back after the holidays.